docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md

```diff
@@ -0,0 +1,16 @@
+# How to perform secure Over-the-Air (OTA) updates? (nRF 9160)
```

Comment:
There is no such thing as `nRF 9160`. It's either `nRF9160` (which means the chip itself) ot `nRF9160 DK` (which means the Development Kit with the nRF9160 chip).

Reply:
Still, not every occurrence has been changed.